### PR TITLE
feat: Add articles pagination

### PR DESCRIPTION
**Pull request by ivan-badyulya**
Added pagination for articles page

JIRA - https://artsyproduct.atlassian.net/browse/PX-3828

Screenshot:
Screen Shot 2021-04-06 at 6 13 43 PM

### CODE REVIEW

File: src/schema/v2/partner.ts [hunk 1]
```diff
@@ -96,7 +111,9 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
             limit: number
             count: boolean
             offset: number
-            sort: string
+            sort?: ArticleSort
+            page: number
```
Comment (sweir27) [inline src/schema/v2/partner.ts:115]: Does the endpoint we're querying accept `page` and `size` params? I'm not super familiar with the articles API, but my understanding from [this code](https://github.com/artsy/positron/blob/5aa2670bf7ca053e3c2dd9770f6fd257cabc6ab6/src/api/apps/articles/model/retrieve.js) was that it only accepts `limit` and `offset`.

**Review by sweir27 — COMMENTED** [review]
Couple of questions!

File: src/schema/v2/sorts/article_sorts.ts [hunk 2]
```diff
@@ -1,17 +1,21 @@
 import { GraphQLEnumType } from "graphql"
 
+export const ARTICLE_SORTS = {
+  PUBLISHED_AT_ASC: {
+    value: "published_at",
+  },
+  PUBLISHED_AT_DESC: {
+    value: "-published_at",
+  },
+} as const
```
[the remark this answers is not in the document]
Comment (ivan-badyulya) [reply src/schema/v2/sorts/article_sorts.ts:10]: I`m not totally sure as well, so I did it based on artists_sorts.ts example in any case

[the remark this answers is not in the document]
Comment (ivan-badyulya) [reply src/schema/v2/sorts/article_sorts.ts]: fixed

File: src/schema/v2/partner.ts (hunk 1 above)
Comment (ivan-badyulya) [reply src/schema/v2/partner.ts]: Actually the pagination works on cursors only, so we do not need the fields so far. I removed them. Fixed

File: src/schema/v2/partner.ts [hunk 3]
```diff
@@ -105,13 +122,19 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
...
-            sort,
+            sort: args.sort,
+            size,
+            page,
           }
 
-          const { results, count } = await articlesLoader(articleArgs)
+          const { results, count } = await articlesLoader({
+            ...articleArgs,
+            ...omit(args, ["first", "last", "before", "after"]),
```
[the remark this answers is not in the document]
Comment (ivan-badyulya) [reply src/schema/v2/partner.ts]: Yes it works with the only articleArgs as well, I fixed it. Sorry tried to do it as fast as possible, so did based on examples in the repo

**Pull request merged**